tests/unit_tests/test_unit.py

```
@@ -284,6 +289,49 @@ def test_mztab_save(tiny_config, tmp_path):
+
+def test_log_annotate_report_average_precision(monkeypatch):
+    mock_logger = unittest.mock.MagicMock()
+    monkeypatch.setattr("casanovo.utils.logger", mock_logger)
+    monkeypatch.setattr(utils, "log_run_report", lambda **_: None)
+    utils.log_annotate_report([], average_precision=0.75)
+    calls = [
+        c
+        for c in mock_logger.info.call_args_list
+        if "Average Precision" in c[0][0]
+    ]
+    assert calls and calls[0][0][1] == 0.75
```

Ruff (0.16.2)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)